internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcClient.java

```diff
@@ -109,7 +109,7 @@ public ShuffleServerGrpcClient(String host, int port, int maxRetryAttempts, bool
     blockingStub = ShuffleServerGrpc.newBlockingStub(channel);
   }
   
-  private ShuffleServerBlockingStub getBlockingStub() {
+  public ShuffleServerBlockingStub getBlockingStub() {
```

Comment:
Could we use `default access modifier`? It seems enough to use `package-private` .
```
ShuffleServerBlockingStub getBlockingStub()
```

Reply:
ut can't access this method if  use default access modifier

Reply:
ut can't access this method if  use default access modifier

Reply:
OK